lib/vdsm/virt/livemerge.py

```diff
@@ -287,6 +287,14 @@ def merge(self, driveSpec, base, top, bandwidth, job_id):
             except JobExistsError as e:
                 raise exception.MergeFailed(str(e), job=job.id)
 
+            if self._base_needs_prune_bitmaps(drive, job, base_info):
+                # Prune stale bitmaps from base volume before measuring.
```

Comment:
The comment does not add any value, it describe what the code does, which is pretty 
clear.

The only interesting info is "before measuring". This comment can come before the if
and say something like:

    # Prune stale bitmaps if needed once before measuring the image. Current
    # qemu-img ignores bitmaps in base but future version may not.
    if self._base_needs_prune_bitmaps(drive, job, base_info):
        ...